SWLOR.Game.Server/Feature/RecipeDefinition/SmitheryRecipeDefinition/RangedRecipes.cs

```diff
@@ -277,6 +277,24 @@ private void Rifles()
                 .Component("ref_scordspar", 5)
                 .Component("ref_plagionite", 5)
                 .Component("ref_keromber", 5);
+
+            // Bol Rifle
```

Comment:
This is stronger than even the Chiro Rifle. If it's supposed to be on par with the Chiro then please reduce its DMG and give it equivalent additional stats.